### PR TITLE
Parallel.ForEach

### PR DESCRIPTION
**Pull request by thirdppp**
Parallel.ForEach

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -74,11 +74,13 @@ Task("TestUsingExampleImplementation")
     .IsDependentOn("ReplaceStubWithExample")
     .Does(() => {
         var allProjects = GetFiles(buildDir + "/*/*.csproj");
-        Parallel.ForEach(allProjects, parallelOptions, (project) => DotNetCoreTest(project.FullPath));
+       Parallel.ForEach(refactoringProjects, (project) => {
```
Comment (robkeim) [inline build.cake]: Hello @thirdppp! Could you give us some more context on what problem you're trying to solve here?

@ErikSchierboom knows the build system better than I do, but my understanding is that `DotNotCoreTest` now contains `DotNetCoreBuild` so it's not necessary to explicitly call it.

Comment (jpreese) [reply build.cake]: @thirdppp , @robkeim is correct in this case. DotNetCoreTest() will call the build for us so there's no need to put it into the ForEach as we'd be building the solution twice.

Comment (ErikSchierboom) [reply build.cake]: Note: this is a feature of the .NET Core version 2.0, it doesn't work with version 1.x

**Comment by jpreese** [comment]
@thirdppp I'm going to close this due to inactivity. There's a ton of other opportunities in this repository if you would like to still contribute!

**Pull request closed without merging**